test/jdk/java/net/URLPermission/URLPermissionTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2013, 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2013, 2016, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello Ryan, this should be `2013, 2022,` - we only maintain 2 years the year this file was first introduced and the latest update year.

Reply:
Good catch!

Reply:
updated, thanks Jaikiran